MMDevice/MMDevice.h

```diff
@@ -603,6 +603,15 @@ namespace MM {
       virtual int SetOrigin() = 0;
       virtual int GetLimits(double& lower, double& upper) = 0;
 
+      /**
+      * @brief Stages can use the OnStagePositionChanged callback to signal
+      * updates about their position.  Some adapters do so, others do not,
+      * in which case the UI code should use polling.  This function signals whether
+      * the device adapters uses callbacks, so that the UI knows it does not need
+      * to poll this device
+      */
+      virtual bool UsesOnStagePositionChanged() = 0;
```

Comment:
just curious: how do you intend to determine from the UI whether a specific device returns True on this... does it imply that you intend to add another public method to MMCore?

Reply:
Very good point!  Completely forgot about that part.  Added now.  Please feel free to suggest better function names.